packages/docusaurus-theme-search/package.json

```
@@ -0,0 +1,38 @@
+{
+  "name": "@markprompt/docusaurus-theme-search",
+  "version": "0.0.0",
+  "description": "A Markprompt search theme for Docusaurus",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/motifland/markprompt-js.git",
+    "directory": "packages/react"
+  },
+  "license": "MIT",
+  "sideEffects": false,
+  "type": "module",
+  "exports": "./dist/index.cjs",
+  "main": "dist/index.cjs",
+  "files": [
+    "dist",
+    "src/theme"
+  ],
+  "scripts": {
+    "prepack": "tsc --build && cpy 'src/**/*.css' dist"
+  },
+  "dependencies": {
+    "@docusaurus/types": "^2.0.0",
+    "@markprompt/core": "0.4.2",
+    "@markprompt/react": "0.3.1",
```

Maybe this would be ok until we hit `1.0.0`?

```suggestion
    "@markprompt/react": "~0.3.1",
```

---

I understand. A major version of `0` is a bit of a special case.